src/screens/DiscoveryScreenV2/index.js

```diff
@@ -21,12 +22,12 @@ import {
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
-
-// const { height } = Dimensions.get('screen');
+import FollowingAction from '../../context/actions/following';
+import {Header} from '../../components';
 
 const DiscoveryScreenV2 = ({route}) => {
   const {tab} = route.params;
-  const [selectedScreen, setSelectedScreen] = React.useState(tab);
+  const [selectedScreen, setSelectedScreen] = React.useState(tab || 0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default value of `selectedScreen` state has been changed from `tab` to `tab || 0`. This change is good as it provides a fallback value (`0`) when `tab` is `undefined` or `null`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->